_posts/Prakash-sa/2022-10-06-clicky-turtle-hack.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: Prakash-sa
+title: Clicky Turtle Hack
```

Comment:
This will work because there are no special characters, but I'd recommend using quotes every time regardless.

Reply:
I have added it now.